packages/xo-web/src/xo-app/hub/resource.js

```diff
@@ -103,7 +107,16 @@ export default decorate([
                   pool.default_SR
                 ),
               })
-            )
+              const oldTemplate = find(
```

Comment:
We should probably delete all of them if there are more than one.

Reply:
But it can never be more than one !

Reply:
In a normal workflow, but nothing ever stays in the normal workflow. So IMO, it doesn't cost much to be safe and clean them all.

Reply:
done :heavy_check_mark: